contrib/completion/zsh/_docker

```diff
@@ -457,7 +457,7 @@ __docker_subcommand() {
         "($help -a --attach)"{-a=,--attach=}"[Attach to stdin, stdout or stderr]:device:(STDIN STDOUT STDERR)"
         "($help)*--add-host=[Add a custom host-to-IP mapping]:host\:ip mapping: "
         "($help)--blkio-weight=[Block IO (relative weight), between 10 and 1000]:Block IO weight:(10 100 500 1000)"
-        "($help)--blkio-weight-device=-[Block IO (relative device weight)]:device:Block IO weight: "
+        "($help)--blkio-weight-device=[Block IO (relative device weight)]:device:Block IO weight: "
```

Comment:
If `--blkio-weight-device` can be specified multiple times, you need to add a `*` in front of `--blkio-weight-device`.

```
"($help)*--blkio-weight-device=[Block IO (relative device weight)]:device:Block IO weight: "
```